hypha/apply/dashboard/templates/dashboard/applicant_dashboard.html

```diff
@@ -34,8 +34,8 @@ <h2 class="font-light flex-1">{% trans "My active submissions" %}</h2>
                         {% for submission in my_active_submissions %}
                             <div class="wrapper wrapper--status-bar-outer">
                                 <div class="wrapper wrapper--status-bar-inner">
-                                    <div class="mt-5 ml-4">
-                                        <h4 class="heading mb-2 font-bold"><a class="link link--underlined" href="{% url 'funds:submissions:detail' submission.id %}">{{ submission.title }}</a></h4>
+                                    <div class="mt-5 ml-4 lg:max-w-[30%]">
+                                        <h4 class="heading mb-2 font-bold line-clamp-2 hover:line-clamp-none"><a class="link link--underlined" href="{% url 'funds:submissions:detail' submission.id %}">{{ submission.title }}</a></h4>
```

Comment:
```suggestion
                                        <h4 class="heading mb-2 font-bold line-clamp-3 hover:line-clamp-none"><a class="link link--underlined" href="{% url 'funds:submissions:detail' submission.id %}">{{ submission.title }}</a></h4>
```

Let's be more forgiving on the number of lines displayed. Bump up to 3 lines.